scenarios/detection/11_exploring_hyperparameters_on_azureml.ipynb

```diff
@@ -41,14 +41,17 @@
   {
```

Comment:
<p>can we delete these commented out lines:</p><p> </p><p># Get accuracy on test set at IOU=0.5</p><p>#e = detector.evaluate()</p><p>#coco_eval = e.coco_eval["bbox"].eval["precision"]</p><p>#precision_settings = (0, slice(0, None), slice(0, None), 0, 2)</p><p>#acc = np.mean(np.mean(coco_eval[precision_settings]))</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/ComputerVision/pull/356/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
deleted